src/styles/blocs/card.scss

```diff
@@ -0,0 +1,106 @@
+.cards__card {
+  margin-bottom: 46px;
+  margin-right: $card-margin;
+}
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  width: #{$card-size};
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin-bottom: 40px;
+    display: flex;
+    align-self: center;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__name {
+    line-height: 18px;
+    font-weight: 500;
+    font-size: 12px;
+    color: #060b35;
+    margin-bottom: 4px;
+  }
+  &__id-product {
+    @extend %standart-text;
+    margin-bottom: 17px;
+  }
+
+  &__price-value {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-color-grey;
+  }
+
+  &__price-count {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $text-color-dark-grey;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__buttom {
+    display: flex;
+    box-sizing: border-box;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+
+  }
+  &__buttom-link {
+    display: block;
+    text-decoration: none;
+    color: #fff;
+    padding: 12px 56px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    background: #00acdc;
+    border-radius: 5px;
+    transition: 0.4s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties